st2common/st2common/models/system/action.py

```diff
@@ -86,6 +88,18 @@ def get_timeout(self):
     def get_cwd(self):
         return self.cwd
 
+    def _get_env_vars_export_string(self):
+        if self.env_vars:
+            exports = ' '.join(
+                '%s=%s' % (quote_unix(k), quote_unix(v))
```

Comment:
AFAICT quote_unix will run on exports later when the complete command is prepared. Is it still necessary to quote here?